docs/themes/manage_themes.rst

```diff
@@ -104,6 +104,31 @@ To preview a Theme:
 
 .. vale off
 
+Hiding a Theme
+**************
+
+.. vale on
+
+Users can't remove default Themes, but they can hide them from the list of Themes in the Email and Landing Page builders.
+
+To hide a Theme:
+
+1. Log in to Mautic.
+
+2. Click the **Settings** icon.
+
+3. Click **Themes**.
+
+4. In the Themes section, locate the Theme that you want to hide.
+
+5. Select the drop-down before the Theme, and click **Hide**.
+
+6. Mautic moves the Theme to the bottom of the table and show it in grey. This means the Theme is hidden in the builders.
+
+To revert this change open the context menu of the hidden Theme and click **Unhide**.
```

Comment:
This should really be 'reveal' or something like that - unhide isn't really a great term to use. Is this something we have in Core already @escopecz ? If so we can wrap this line in a vale off/on statement.

Reply:
Yes, it's in the core already